dockerImages.yml

```diff
@@ -8,20 +8,22 @@
 
 # Tools are sorted alphabetically for ease of use.
 
-fastqsplitter: "quay.io/biocontainers/fastqsplitter:1.1.0--py37h516909a_1"
 biopet-scatterregions: "quay.io/biocontainers/biopet-scatterregions:0.2--0"
 # A mulled container is needed to have both picard and bwa in one container.
 # quay.io/biocontainers/mulled-v2-002f51ea92721407ef440b921fb5940f424be842
 # This container contains: picard (2.18.7), bwa (0.7.17-r1188)
 "bwa+picard": "quay.io/biocontainers/mulled-v2-002f51ea92721407ef440b921fb5940f424be842:43ec6124f9f4f875515f9548733b8b4e5fed9aa6-0"
 cutadapt: "quay.io/biocontainers/cutadapt:2.4--py37h14c3975_0"
 fastqc: "quay.io/biocontainers/fastqc:0.11.7--4"
+debian: "debian@sha256:f05c05a218b7a4a5fe979045b1c8e2a9ec3524e5611ebfdd0ef5b8040f9008fa"  # Used for general tasks
```

Comment:
Right now this will break the singularity prepull script, as it won't be able to find the digest for the tag `f05c05a218b7a4a5fe979045b1c8e2a9ec3524e5611ebfdd0ef5b8040f9008fa`, but that should probably be fixed in the script rather than here.

Reply:
You could check for the `@` in the line and then pull it as is.